templates/devops_fargate.json

```
@@ -65,15 +65,24 @@
             ]
         },
         {
-            "Sid": "ProviderPermissions",
+            "Sid": "CreateProviderPermissions/OIDCProvider",
```

I don't think `/` are allowed characters. Let's just simplify it.

```suggestion
            "Sid": "CreateGetTagOIDCProvider",
```